frontend/src/app/globals.css

```
@@ -613,6 +613,15 @@ button:enabled,
   --sidebar-accent-foreground: #2c4a7c;
   --sidebar-border: #e8ebef;
   --sidebar-ring: #4285f4;
+  --sidebar-width: 20rem;
+  --sidebar-width-icon: 3rem;
+}
+
+/* Disable transitions during sidebar drag resize for smooth performance */
+.sidebar-resizing,
+.sidebar-resizing * {
+  transition-duration: 0s !important;
+  transition: none !important;
```

Stylelint (17.13.0)

[error] Overridden property "transition-duration" by shorthand "transition" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)